script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -444,14 +445,14 @@ Their 'sickness unto death', if you will.
 [sha:8e000d16dd27fa6d971fe70ba7e68aa814a05179]{
 -- Page 68, Offset 29690.
 -- 「それは理性で我慢をする、といったレベルの話ではありません。具体的、物理的に、彼らは持てるすべての力を使って、常に自分自身の欲求を封印している。
-"It's not just a matter of exercising self restraint through reason.
+"It's not just a matter of exercising self-restraint through reason.
  It takes concrete, physical effort―――every drop of power they have―――to keep that desire sealed away.
 }
 [sha:98a26d80207a9f506cab4e0d3ef3e9802871a999]{
 -- Page 68, Offset 29691.
 -- 強大な異能を自身に使う事で、彼らは吸血衝動を抑えている。けれどもし―――何等かの外的要因でその真祖の能力が低下してしまった場合、抑えていた吸血衝動がどうなるか、分かりますか？」
-By turning their own supernatural abilities on themselves, they're able to suppress their innate blood-sucking urges.
- However―――what do you think would happen to those urges if a True Ancestor's power were to suddenly deteriorate for reasons outside their control?"
+By turning their own supernatural abilities on themselves, they're able to suppress their innate vampiric impulses.
+ However―――what do you think would happen to those impulses if a True Ancestor's power were to suddenly deteriorate for reasons outside their control?"
```

Comment:
revert impulses -> urges on this line. too repetitive with two "impulse"s in a row